apps/modernization-ui/src/apps/search/investigation/GeneralSearchFields.tsx

```diff
@@ -230,7 +255,9 @@ const GeneralSearchFields = () => {
                                         required={true}
                                         onBlur={onBlur}
                                     />
-                                    {error && <ErrorMessage id={`${error}-message`}>{error?.message}</ErrorMessage>}
+                                    {error && (
+                                        <ErrorMessage id={`provider-error-message`}>{error?.message}</ErrorMessage>
+                                    )}
```

Comment:
The autocomplete components should support the `error` property.  I know you just fixed the id but it should not have been like this to begin with